### PR TITLE
Improved message when save lua VTX file

### PR DESCRIPTION
**Pull request by HThuren**
Improve message when save LUA file with VTX configuration

Require 2 more strings in Crowdin (here with danish translation)

    "vtxSavedLuaFileOk": {
        "message": "VTX konfiguration <span class=\"message-positive\">gemt</span> i lua fil",
        "description": "Message in the GUI log when the VTX Config lua file is saved"
    },
    "vtxSavedLuaFileKo": {
        "message": "<span class=\"message-negative\">Fejl</span> ved gemning af lua VTX konfigurations fil",
        "description": "Message in the GUI log when the VTX Config lua file saving has failed"
    },

### CODE REVIEW

**Comment by HThuren** [comment]
Not sure why Linux compliation failed, win64 debug compilation was ok locally

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — DISMISSED** [review]

**Comment by HThuren** [comment]
Right, mcgivergim told the new strings automatically will be added. But maybe first after merge with master (?)

**Comment by McGiverGim** [comment]
Of course, it will not be uploaded to Crowdin until the code it's at master branch.

**Review by blckmn — APPROVED** [review]

**Review by McGiverGim — CHANGES_REQUESTED** [review]
One thing is missing here: you need to add the "English" strings. I thought it existed, but I can't find the two strings, so I suppose this are new strings.
Please add the English version of your strings to the PR.

**Comment by HThuren** [comment]
English version
    "vtxSavedLuaFileOk": {
        "message": "lua VTX Config file <span class=\"message-positive\">saved</span>",
        "description": "Message in the GUI log when the lua VTX Config file is saved"
    },
    "vtxSavedLuaFileKo": {
        "message": "<span class=\"message-negative\">Error</span> while saving the lua VTX Config file",
        "description": "Message in the GUI log when the lau VTX Config file saving has failed"
    },

**Comment by haslinghuis** [comment]
Please add these strings to `locales/en/messages.json`.

**Comment by HThuren** [comment]
Right, something to learn abut the crowdin integration :-)
en/message.jason uploaded

File: locales/en/messages.json
Comment (McGiverGim) [inline locales/en/messages.json:6228]: The strings are cut, please complete them ;)

**Review by McGiverGim — CHANGES_REQUESTED** [review]

File: locales/en/messages.json
Comment (McGiverGim) [reply locales/en/messages.json:6228]: You need to close the </span> tag

Comment (HThuren) [reply locales/en/messages.json:6228]: Missing \ lost in copy paste

File: locales/en/messages.json
Comment (Asizon) [inline locales/en/messages.json:6227]: wrong indendation

**Review by Asizon — DISMISSED** [review]

File: locales/en/messages.json
Comment (McGiverGim) [inline locales/en/messages.json:6224]: You need to put the opening span before the text you want to change the color, and the closing span after it.

File: locales/en/messages.json
Comment (McGiverGim) [inline locales/en/messages.json:6223]:
```suggestion
        "message": "lua VTX Config file <span class=\"message-positive\">saved</span>",
```

**Review by McGiverGim — CHANGES_REQUESTED** [review]

File: locales/en/messages.json
Comment (McGiverGim) [reply locales/en/messages.json:6226]:
```suggestion
    "vtxSavedLuaFileKo": {
```

**Review by McGiverGim — CHANGES_REQUESTED** [review]

**Comment by HThuren** [comment]
hi @McGiverGim and @Asizon, sorry for so much effort from you regarding this small change.

**Review by McGiverGim — APPROVED** [review]

**Review by haslinghuis — APPROVED** [review]

**Comment by HThuren** [comment]
thank you

**Pull request merged**
